Vsix/Vsix.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```

Comment:
Purely out of interest, what method/tool did  you use to update things that caused this?

Reply:
visual studio 2022, created a new Vsix project because it did not compile for some reason

Reply:
Interesting, I use Vs2022 with no problem. Could you let me know what the compile error was?